src/Administration/Dashboard/OrderAmountService.php

```diff
@@ -43,7 +43,7 @@ public function load(Context $context, string $since, bool $paid, string $timezo
             'DATE_FORMAT(' . $accessor . ', \'%Y-%m-%d\') as `date`',
             'COUNT(`order`.id) as `count`',
             'SUM(
-                `order`.amount_total / IFNULL(`order`.currency_factor, currency.factor)
+                `order`.amount_total * IFNULL(`order`.currency_factor, currency.factor)
```

Comment:
Nevermind, I got this wrong and division is actually correct here at least for `currency.factor`.

Just double checking if I got this correctly. The `currency.factor` is always a factor and thus should be multiplied against the main currency with factor `1`. e.g. see admin settings:
![image](https://github.com/user-attachments/assets/d7bf12ff-9654-4a42-a06a-bde119d058e5)

If you have an order in "Swedish krona" of 100, that needs to be divided by `10.51` to get back `9.51`€. This matches what google says for the conversion.

What I'm unsure about is if this `order.currency_factor` works in the same way (division) or is that a multiplier? 🤔

Reply:
Seems like `order.currency_factor` is the same as `currency.factor` at the time the order was created.

Turns out only factors below `1` are causing issues. E.g. `100 NOK / 0.099` is `1 010.10€` which is wrong and instead should be `9.90 NOK`. A have to think about this a bit more.... 🙈

Reply:
Looks like the factor actually represents 

> 1 EUR = X units of currency (factor of that currency)
> e.g. 1 EUR = 0.89 GBP or 1 EUR = 1.17 USD

Which also means at least the factor for NOK is wrong as that follows the assumption of
> how much 1 unit of that currency is worth in the base currency

Which is a totally different approach. Depending on the approach you have to multiply or divide